http-clients/src/test/java/org/triplea/http/client/moderator/toolbox/ModeratorToolboxFeignClientTest.java

```diff
@@ -0,0 +1,202 @@
+package org.triplea.http.client.moderator.toolbox;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static java.util.Arrays.asList;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.hamcrest.core.Is.is;
+
+import java.net.URI;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+
+import javax.annotation.Nonnull;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.triplea.http.client.HttpClient;
+import org.triplea.http.client.HttpClientTesting;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+
+import lombok.Builder;
+import ru.lanwen.wiremock.ext.WiremockResolver;
+import ru.lanwen.wiremock.ext.WiremockUriResolver;
+
+@ExtendWith({
+    WiremockResolver.class,
+    WiremockUriResolver.class
+})
```

Comment:
This is probably a design preference, but for me
```java
@ExtendWith(WiremockResolver.class)
@ExtendWith(WiremockUriResolver.class)
```
seems more logical?

Regardless I'm not sure if this is done in a consistent manner, so that might be a goal.

Reply:
I forget that Java now allows the same annotation multiple times. IMO this is a good cleanup to do post-merge, the annotation for now is consistent with other client tests.

Reply:
Hmm, trying out that cleanup:
```java
@ExtendWith(WiremockResolver.class)
@ExtendWith(WiremockUriResolver.class)
```
I'm actually not sure I like it. When duplicating the `@ExtendWith`, the two are not guaranteed to be together and they should/must be. I think unrelated `@ExtendWith` are probably better to be done as additional annotations.

Initially my preference seems to be for something like:

```java
@ExtendWith({
    WiremockResolver.class,
    WiremockUriResolver.class
})
@ExtendWith(MockitoExtension.class)
```

rather than:
```java
@ExtendWith(WiremockResolver.class)
@ExtendWith(WiremockUriResolver.class)
@ExtendWith(MockitoExtension.class)
```

I think we're really just getting down to personal preference here, probably best we revisit this in a follow-up PR if someone is really inclined to change it and/or add a note to the coding styling guide.